test/Generator.V3.Tests/AssertHelper.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Are these helper files same for both v2 and v3 unit test projects? If yes, we can have a single copy in a common project. Let me know your thoughts on this.

Reply:
As discussed, there are only 2 helper files and moving them to the Generator Project would not be correct as these are related to unit test cases. Hence for now keeping them here.